meinberlin/apps/dashboard/urls.py

```diff
@@ -38,4 +38,11 @@
     url(r'^organisations/(?P<organisation_slug>[-\w_]+)/projects/$',
         views.DashboardProjectListView.as_view(),
         name='project-list'),
+    url(r'^projects/(?P<project_slug>[-\w_]+)/blueprints/$',
```

Comment:
we should move those to url to the top and rename the block from "# Overwrite the ProjectUpdateView with meinBerlin urls" to "# Overwrite adhocracy4 core urls with meinBerlin urls"